src/config/routes.rb

```diff
@@ -9,8 +9,14 @@
   delete '/sign_out', to: 'sessions#destroy', as: :sign_out
 
   get '/course', to: 'course#index', as: :course
-  get '/quiz', to: 'quiz#index', as: :quiz
+
   get '/quiz/new', to: 'quiz#new', as: :new_quiz
```

Comment:
There is a short version of doing this.

http://guides.rubyonrails.org/routing.html#crud-verbs-and-actions